common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -33,6 +33,6 @@
     <div class="script_placeholder" data-src="${previewer}"/>
 
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-describedby="label_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
Might as well also link "status" on line 19 to label_${id} via aria-describedby.